pkg/kubelet/network/dns/dns.go

```diff
@@ -349,7 +357,9 @@ func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
 				dnsConfig.Servers = append(dnsConfig.Servers, ip.String())
 			}
 			dnsConfig.Searches = c.generateSearchesForDNSClusterFirst(dnsConfig.Searches, pod)
-			dnsConfig.Options = defaultDNSOptions
+			glog.Info("try to append %s with %s", dnsConfig.Options, defaultDNSOptions[0])
+			dnsConfig.Options = mergeDNSOptions(dnsConfig.Options, nil, defaultDNSOptions)
+			glog.Infof("dnsConfig.Servers: %s, dnsConfig.Searches: %s, dnsConfig.Options: %s", dnsConfig.Servers, dnsConfig.Searches, dnsConfig.Options)
```

Comment:
would these two glog.Info() calls spam the logs a lot?
would it be better to add verbosity level using e.g. `V(2)`?